### ISSUE TITLE
snowboy support on Jetson Nano

### ISSUE THREAD

**Issue by xtendDemo** [Kitt-AI/snowboy#684]
hello,
Snowboy can be installed on Jetson Nano aarch64 Ubuntu 18.04?

**Comment by pzarzycki** [comment]
Yes, easily. Tested with C++ client. You might however struggle with generating your own models, since Ubuntu 16.04 is only supported. Is this project slowly dying?

**Comment by Aculeasis** [comment]
> You might however struggle with generating your own models, since Ubuntu 16.04 is only supported.

Model generator require x86_64. There is no way to run it on aarch64.

Also, I successfully ran it in [ubuntu 20.04](https://github.com/Aculeasis/snowboy-pmdls/blob/master/Dockerfile).
